### PR TITLE
be keen to newcomers

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: create_framework/front_controller.rst [hunk 1]
```diff
@@ -61,8 +61,8 @@ which name is exposed to the end user via the URL
 (``http://127.0.0.1:4321/bye.php``): there is a direct mapping between the PHP
 script name and the client URL. This is because the dispatching of the request
 is done by the web server directly. It might be a good idea to move this
-dispatching to our code for better flexibility. This can be easily achieved by
-routing all client requests to a single PHP script.
+dispatching to our code for better flexibility. This can be achieved byrouting
```
Comment (kunicmarko20) [inline create_framework/front_controller.rst]: space missing

Comment (OskarStark) [reply create_framework/front_controller.rst]: thanks, done

**Comment by OskarStark** [comment]
This PR is ready to review, it would grow too much

I will open another following PR if this is done.

**Comment by javiereguiluz** [comment]
Thanks Oskar! I've merged this one to avoid having a too big and too difficult to merge pull request. We can merge PRs like this in the future to fix more issues.

**Pull request merged**